runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -171,6 +183,32 @@ public void clearAll() {
         corfuTable.clear();
     }
 
+    /**
+     * Allow GC to remove all the in-memory objects associated with the CorfuTable
+     * and make it look like the table is a newly opened one
+     * This is useful for JVMs who do not wish to keep the entire table
+     * around in memory.
+     * @param runtime - the runtime that was used to create this table
+     * @param serializer - the serializer used to materialize table data
+     */
+    public void resetTableData(CorfuRuntime runtime, ISerializer serializer) {
+        ObjectsView.ObjectID oid = new ObjectsView.ObjectID(getStreamUUID(), CorfuTable.class);
+        Object tableObject = runtime.getObjectsView().getObjectCache().remove(oid);
+        if (tableObject == null) {
+            throw new NoSuchElementException("closeTable: No object cache entry for "+ fullyQualifiedTableName);
```

Comment:
nit -> closeTable -> resetTableData (exception message)

Reply:
done, thanks for catching this